queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/RequestCommands/PosReceiptCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts.Constants;
+using fiskaltrust.Middleware.Contracts.Repositories;
+using fiskaltrust.Middleware.Contracts.RequestCommands;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.RequestCommands
+{
+    public struct RefundDetails
+    {
+        public string Serialnumber { get; set; }
+        public long ZRepNumber { get; set; }
+        public long ReceiptNumber { get; set; }
+        public DateTime ReceiptDateTime { get; set; }
+    }
+
+    public class PosReceiptCommand : RequestCommand
+    {
+        private readonly long _countryBaseState;
+        private readonly ICountrySpecificQueueRepository _countrySpecificQueueRepository;
+        private readonly ICountrySpecificSettings _countryspecificSettings;
+        private readonly IConfigurationRepository _configurationRepository;
+
+        public PosReceiptCommand(IConfigurationRepository configurationRepository, ICountrySpecificSettings countrySpecificSettings)
+        {
+            _countryspecificSettings = countrySpecificSettings;
+            _countrySpecificQueueRepository = countrySpecificSettings.CountrySpecificQueueRepository;
+            _countryBaseState = countrySpecificSettings.CountryBaseState;
+            _configurationRepository = configurationRepository;
+        }
+
+        public override Task<RequestCommandResponse> ExecuteAsync(ftQueue queue, ReceiptRequest request, ftQueueItem queueItem, bool isBeingResent = false)
+        {
+            return Task.FromResult(new RequestCommandResponse
```

Comment:
So IMO we should change this and adapt it to be closer to what we have in germany e.g.

https://github.com/fiskaltrust/middleware/blob/29b8170ffbfe64a528337a5fdcc754d37a6653e2/queue/src/fiskaltrust.Middleware.Localization.QueueDE/RequestCommands/RequestCommand.cs#L168

Reply:
You mean don't create the `ReceiptResponse` inline but introduce a `CreateReceiptResponse` method that handles that?

Reply:
Well no  not really. More or less we should use the one from the base class and augement it if necessary. :)